neuralprophet/time_net.py

```diff
@@ -102,14 +105,19 @@ def __init__(
         # General
         self.n_forecasts = n_forecasts
 
+        # Quantiles
+        self.quantiles = quantiles
+
         # Bias
-        self.bias = new_param(dims=[1])
+        # dimensions - [no. of quantiles, bias shape]
```

Comment:
maybe we should have the quantiles always as the last dimension?

Reply:
Per our discussion, quantiles will only do after batch and no. forecasts.